css/style.css

```diff
@@ -1,19 +1,314 @@
+:root {
+  --grey-dark: #616364;
+  --grey-light: #a7a4a4;
+  --orange-dark: #c05326;
+  --orange-light: #f7eae4;
+  --orange: #d65a17;
+  --white: #fff;
+  --black: black;
+  --pink: #f7d1b980;
+}
+/*Body*/
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  max-width: 100vw;
+  margin: 0 auto;
+}
+
+h1,
+h2,
+h4,
+h5,
+h6,
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* Header */
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+header {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+  margin: 2px 90px;
```

Comment:
Hi Zeliha, I am writing this comment for my "Module Project: Code Review" ticket. Great work overall, well done. I just wanted to say if you would use padding here instead of margin, you could implement your background color to your entire header. Background color changes the color of element until border, it doesn't effect margins color. But except this issue, great web site.